cloud_build/deploy.sh

```diff
@@ -6,5 +6,5 @@
 # Deploy a new flutter dashboard version to google cloud.
 
 pushd app_dart > /dev/null
```

Comment:
use set -e at the top of these scripts as they are they will continue even if one command fails.

Reply:
Added.